### PR TITLE
Heman/generate 1 SKU pack

### PR DESCRIPTION
**Pull request by hemangandhi**
Decided to start merging my branches from here.

All the extra parameters are from the orchestrator settings (particularly `toolname` (now `toolName` for consistency with the validation API) and `systemState['locationName']` and `systemState['partType']`).

The `patternName` is also passed so I can override it after merging the parameters -- I think planning is the best place for the merge since it has the OpenRAVE enviroment units.

### CODE REVIEW

File: python/mujinplanningclient/packingplanningclient.py [hunk 1]
```diff
@@ -56,17 +60,30 @@ def StartPackFormationComputationThread(self, timeout=10, debuglevel=4, toolname
         taskparameters.update(kwargs)
         return self.ExecuteCommand(taskparameters, toolname=toolname, timeout=timeout)
 
-    def StartSingleSkuPackFormationComputationThread(self, packingParameters, packLocationName, toolname=None, timeout=10, **kwargs):
+    def StartSingleSKUPackFormation(
+        self,
+        userPackFormationComputationParameters,
+        systemState,
+        toolname=None,
```
Comment (ivandkhn) [inline python/mujinplanningclient/packingplanningclient.py:67]: I think we should agree on a consistent capitalization of `toolname`, at least for the clients. In validation API it's `toolName`, even though it's immediately converted to `toolname` when passed down to a packing task.

Comment (hemangandhi) [reply python/mujinplanningclient/packingplanningclient.py]: Sure.

**Review by ivandkhn — APPROVED** [review]

**Review by andriymujin — APPROVED** [review]

File: python/mujinplanningclient/packingplanningclient.py [hunk 2]
```diff
@@ -56,20 +60,33 @@ def StartPackFormationComputationThread(self, timeout=10, debuglevel=4, toolname
...
+    def StartSingleSKUPackFormation(
+        self,
+        userPackFormationComputationParameters,
+        systemState,
+        toolName=None,
+        patternName=None,
+        timeout=None,
+        **kwargs
+    ):
+        # type: (dict[str, Any], dict[str, Any], Optional[str], Optional[str], Optional[float], Any) -> None
```
Comment (felixvd) [inline python/mujinplanningclient/packingplanningclient.py:72]:
```suggestion
    def StartSingleSKUPackFormation(
        self,
        userPackFormationComputationParameters,  # type: dict[str, Any]
        systemState,  # type: dict[str, Any]
        toolName=None,  # type: Optional[str]
        patternName=None,  # type: Optional[str]
        timeout=None,  # type: Optional[float]
        **kwargs  # type: Any
    ):  # type: (...) -> None
```
nit

Comment (hemangandhi) [reply python/mujinplanningclient/packingplanningclient.py]: Interestingly, though `darker` split up the lines, it did not move the types up. I guess modern tooling doesn't understand ancient Python.

Comment (felixvd) [reply python/mujinplanningclient/packingplanningclient.py]: Yes, and who could blame it. Pretty sure it would have worked in Python 3.

**Pull request merged**